doc/developer-guides/hld/hld-devicemodel.rst

```diff
@@ -789,6 +789,137 @@ the bus hierarchy would be:
    00:04.0 Ethernet controller: Red Hat, Inc. Virtio network device
    00:05.0 Serial controller: Red Hat, Inc. Virtio console
 
+.. _PCIe PTM implementation:
+
+PCIE Root Port Emulation
+========================
+
+The PCI Express (PCIe) specification defines a Precision Time Measurement (PTM)
+mechanism that enables time coordination and synchronization of events across
+multiple PCI components with independent local time clocks within the same
+system.  Intel supports PTM on several of its systems and devices, such as PTM
+root capabilities support on Whiskey Lake and Tiger Lake PCIe root ports, and
+PTM device support on an Intel I225-V/I225-LM family Ethernet controller.  For
+further details on PTM, please refer to the `PCIe specification
+<https://pcisig.com/specifications>`_.
+
+ACRN adds PCIe root port emulation in the hypervisor to support the PTM feature
+and emulates a simple PTM hierarchy.  ACRN enables PTM in a Guest VM if the user
+sets the ``enable_ptm`` option when passing through a device to a post-launched
+VM. When the user enables PTM, the passthru device is connected to a virtual
+root port instead of the host bridge.  Here is an example launch script that
```

Comment:
Good point.  But I suppose this has been taken care of already since launch script already has this?  Maybe revisit the document of launch script to add this.

Reply:
It sounds good to me.  But I would like to see whether @gvancuts and @wenlingz have any question after reading above paragraph?